Source/websocket/WebTransfer.h

```diff
@@ -308,9 +308,9 @@ namespace Web {
         {
             return (_fileBody.Position());
         }
-        inline void LoadHashContext(const uint8_t context[])
+        inline void LoadHash(Crypto::Context& context)
```

Comment:
You know I am a bif fan of const ;-) Can we make the context here a const parameter ?

Reply:
done